src/import/engine.rs

```diff
@@ -149,7 +149,6 @@ impl SSTInfo {
         let path = info.file_path();
         let mut f = env.new_sequential_file(path.to_str().unwrap(), EnvOptions::new())?;
         f.read_to_end(&mut data)?;
-        assert_eq!(data.len(), info.file_size() as usize);
```

Comment:
why remove it

Reply:
because the file size of not encrypted file and raw file is not the same, the size of raw file is the size of not encrypted file plus the size of prefix